tests/unit/test_backends.py

```diff
@@ -177,6 +161,53 @@ def getCallSets(self, variantSetId, pageSize=100):
             request, pageSize, self._backend.searchCallSets,
             protocol.GASearchCallSetsResponse, "callSets")
 
+
+class TestSimulatedBackend(TestAbstractBackend):
+
+    def setUp(self):
+        self._backend = backend.SimulatedBackend(
+            numCalls=100, numVariantSets=10)
+        # TODO arbitrary values
+
+    def tearDown(self):
+        pass
+
+
+class TestPaging(TestSimulatedBackend):
+
+    def testVariantSetPagination(self):
+        results = []
+        for pageSize in range(1, 10):
+            variantSetIds = [
+                variantSet.id for variantSet in self.getVariantSets(
+                    pageSize=pageSize)]
+            results.append(variantSetIds)
+        for result in results[1:]:
+            self.assertEqual(result, results[0])
+        # TODO repeat this for full object equality, not just ids
```

Comment:
We might as well just do this for object equality now and get rid of this comment. We can replace this with something like
```python
results = []
for pageSize in range(1, 10):
    results.append(list(self.getVariantSets(pageSize=pageSize)))
for result in results[1:]:
    self.assertEqual(result, results[0])
```